### ISSUE TITLE
Member constraints are not properly processed

### ISSUE THREAD

**Issue by Martin521** [nojaf/telplin#9]
Issue created from [telplin-online](https://nojaf.com/telplin/#data=N4KABGBECWC2AOAbAprZA7ALgQ09A9upAFxSz4AmArimACrKJLToA6b6KmYLiLyYAM5VYYAB6CwAXjAAZaIMwA6YaIntIIAL5A)

#### Implementation

```fsharp
module Telplin

let inline sum xs = List.sum xs

```
#### Signature

```fsharp
module Telplin

val inline sum: xs: ^a list -> ^a

```

#### Problem description

The correct signature should be

```fsharp
val inline sum: xs: ^a list -> ^a when ^a: (static member (+): ^a * ^a -> ^a) and ^a: (static member Zero: ^a)
```

#### Extra information

- [X] The proposed signature has problems.
- [X] I would be willing to help fix this.

**Comment by Martin521** [comment]
I thought this one might be easier than #8, but it doesn't look that way.
The member constraint data, I think, cannot that easily be transported as text in a 'GenericConstraint' as it has been done for the CoercesToConstraint. (I mean transported from the typed tree of the implementation to `mkSynTypForSignatureBasedOnTypedTree`.)
Do you have any thoughts already on how to proceed here?

**Comment by Martin521** [comment]
Looking at #10, it seems I was on the right track actually, I just thought there needed to be a simpler solution. I learned a lot while investigating.
I am closing this one, as it was fixed by #10.
